### PR TITLE
Building todo list app

### PR DESCRIPTION
**Pull request by daviidy**
For this application, the user can:
- add new projects
- add tasks to a project
- edit the task
- show details of the task
- delete the task
We created two classes:
- one for project objects
- one for task objects
We added some modules:
- for adding a new project
- for adding a new project to the DOM
- for showing a new project
- for showing a new project to the DOM
- for adding a new task
- for adding a new task to the DOM
- for showing a new task
- for deleting a task

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: Invalid :red_circle: 

Hi, @team, your PR is invalid cause your project is not working on localhost also in the remote live demo, please next time always check locally if everything is working then ask for a code review, to save your own time and also to save reviewer effort.

I have left some comments below kindly check them:
:point_down: :point_down: :point_down: :point_down: 

## Required changes:

- [ ] Change the title of the readme.
- [ ] remove one extras have(readme):
image

## Optional Changes:

- [ ] add these scripts to your package.json file, and update your readme on how to run the project:
```
  "scripts": {
    "start": "webpack --watch",
    "build": "webpack --mode=production",
    "server": "webpack-dev-server --open"
  },

```
_don't forget to install webpack-dev-server package_

and make sure you are using these versions on your package.json
```
    "webpack": "^4.44.2",
    "webpack-cli": "^3.3.12",
    "webpack-dev-server": "^3.11.0"
```

### Please make all the required changes before you ask for another code review.
## Happy Coding :tophat: 
_Feel free to contact me on slack if you have any questions or need help_
## Zakariae El Mejdki.

**Comment by sinework** [comment]
Hello there, 
I'm not sure how this is happening but the feature-branch (the branch we are working on) is working fine on multiple machines and the live demo is active as well.  Can you please share with us the error you are getting?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "This is the second project in the Microverse Javascript module. The goal of the project is to learn basic webpack, ES6 modules and classes.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (BrittanyBlake) [inline package.json]: - [x] Scripts should include all of the necessary run commands like 👇🏾 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @daviidy and @sinework ! 👋 
You have done a really great job with this project! It looks really good! 🎖️ 🤩 There are just a few adjustments to make before you can move on 👇🏾 
****

- [ ] See my comments below. 

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by sinework** [comment]
Thank you, we have made the required change.

**Review by Forison — APPROVED** [review]
# Status::Approved✔️.
### Good job on addressing the issue raised in the previous review, kindly go ahead and merge the branch and proceed to the next milestone🤗. 
#### Happy coding
Addo Forison

**Pull request merged**